tgui/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "isolatedModules": false
+    "isolatedModules": false,
+    "baseUrl": "packages"
```

Comment:
This change is to allow the aliased package (`common` is the one this PR uses, but would apply to others) be available to TypeScript. The `bem.js` file was converted to `bem.ts`.

I am not convinced this is the "right" way of doing this, will need to investigate further, so don't merge until it's been looked at a bit I guess.

@stylemistake @ZeWaka

Reply:
/shrug, i think it worked without it.

Reply:
Possibly! I'll see if I would get the same warning in tgstation's repo. It certainly "works" in either case, just a few of the VS Code bits grump (Problems, and a warning).